docs/critical.md

```diff
@@ -530,7 +530,7 @@ ad lumen solis
 
 This would make it easy to create the following apparatus entry:
 
-> nicticoracis *post* oculus A
+> nicticoracis *post* oculus *B*
```

Comment:
Why change B to *B* while changing below the *A* to A. Shouldn't this be "B", without italics.